src/installer/exceptions.py

```diff
@@ -0,0 +1,27 @@
+__all__ = [
+    "InvalidWheel",
+    "MetadataNotFound",
+    "RecordItemError",
+    "RecordItemHashMismatch",
+    "RecordItemSizeMismatch",
+]
+
+
+class InvalidWheel(Exception):
+    pass
```

Comment:
instead of pass consider adding a docstring that explains when the error may be thrown